apps/app/src/app/pages/(components)/components/(select)/select.page.ts

```diff
@@ -78,6 +80,13 @@ export const routeMeta: RouteMeta = {
 				</div>
 				<spartan-code secondTab [code]="scrollableCode" />
 			</spartan-tabs>
+			<h3 id="examples__header_footer" class="${hlmH4} mb-2 mt-6">Header and Footer Content</h3>
```

Comment:
Could you please add here an additional info please. 

Qoute from sam in the last PR

> but we might want to mention in docs that adding anything to complex in these areas might not be accessible or affect accessibility

Reply:
Added comment advising for not using complex ui components and making considerations for accessibility